test/logflare_web/live_views/log_event_live_test.exs

```diff
@@ -0,0 +1,101 @@
+defmodule LogflareWeb.LogEventLiveTest do
+  @moduledoc false
+  use LogflareWeb.ConnCase
+  alias Logflare.Logs.LogEvents
+
+  import ExUnit.CaptureLog
+
+  setup %{conn: conn} do
+    insert(:plan)
+    user = insert(:user)
+    source = insert(:source, user: user)
+    conn = login_user(conn, user)
+
+    on_exit(fn ->
+      Cachex.clear(LogEvents.Cache)
+    end)
+
+    %{user: user, source: source, conn: conn}
+  end
+
+  test "show by uuid with no timestamp param", %{conn: conn, source: source} do
+    le = build(:log_event, source: source, message: "some message")
+
+    pid = self()
+    ref = make_ref()
+
+    expect(GoogleApi.BigQuery.V2.Api.Jobs, :bigquery_jobs_query, 1, fn _conn, _proj_id, opts ->
+      send(pid, {:query, ref, opts[:body].query})
+
+      {:ok,
+       TestUtils.gen_bq_response([%{"id" => le.id, "event_message" => le.body["event_message"]}])}
+    end)
+
+    {:ok, view, _html} = live(conn, ~p"/sources/#{source.id}/event?#{%{uuid: le.id}}")
+
+    :timer.sleep(1500)
```

Comment:
lets use retry_assert so that the test doesn't take too long as this sets the min time